docs: Fixing custom implementations of Utility APIs (#4563)
backstage/backstage: docs/api/utility-apis.md

this:

```ts
const app = createApp({
  apis: [
    /* ApiFactories */
    createApiFactory(errorApiRef, new IgnoringErrorApi())

    // OR

>     // if you have dependencies inside your additional API you can use the object form

Reviewer: ```suggestion
    // If your API has dependencies, you can use the object form
```
